src/time_integration/paired_explicit_runge_kutta/methods_PERK2.jl

```diff
@@ -107,7 +106,7 @@ function compute_PairedExplicitRK2_butcher_tableau(num_stages,
 end
 
 @doc raw"""
-    PairedExplicitRK2(num_stages, base_path_monomial_coeffs::AbstractString,
+    PairedExplicitRK2(num_stages, base_path_monomial_coeffs::AbstractString, dt_opt,
                       bS = 1.0, cS = 0.5)
```

Comment:
This changed the interface here since it requires an additional argument, so it is a breaking change. Is this stuff declared as experimental so that we can just do it?

Reply:
Yeah, I somehow forgot that I added the breaking label. We should revert this and merge with 0.9.